src/sonic-pit/pit-sysdiag/src/syslogger.py

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env python
```

Comment:
maybe can reuse the existing logger? https://github.com/sonic-net/sonic-buildimage/blob/master/src/sonic-py-common/sonic_py_common/logger.py

Reply:
Syslogger is specially added because we want to control log message format, such as log message header strings(include time, program name, loglevel) and log files name, log file directory, log file size, backlog counts, etc.

So I think maybe we're not replacing the logger.